cli/tests/ld-floxlib/manifest-catalog.toml

```diff
@@ -0,0 +1,14 @@
+version = 1
+
+[install]
+boost.pkg-path = "boost"
+curl.pkg-path = "curl"
+gcc.pkg-path = "gcc"
+gcc-unwrapped = { pkg-path = "gcc-unwrapped", priority = 4 }
+glibc = { pkg-path = "glibc" , version = "2.34-210" }
+libarchive.pkg-path = "libarchive"
+nix = { pkg-path = "nix", priority = 4, version = "2.8.1" }
```

Comment:
question (nb): why do we change the nix version here compared to the pkgdb test form 2.10 to 2.8?

Reply:
There's a mention in the commit message:

> The manifest needed to be forked to include a version and the specific
> versions of glibc and nix that are required by the test. I couldn't
> resolve the original version of nix from the catalog but it seems that
> the specific version is only need to assert on the output, which has
> been updated.

2.10.3 isn't available in the catalog. 2.8.1 seemed to be the only one on the same page as the pinned glibc. Alternatively we could use a newer version like 2.11.0 in a separate `pkg-group`.

Reply:
yeah th epoint was to use an old glibc i guess it doesnt matter too much whether its 2.8 or 2.11 if 2.10 is not available.
just wanted to make sure this was deliberate